### PR TITLE
fix: sd-JWT ref and status assertions ref

### PR DESCRIPTION
**Pull request by peppelinux**
This PR must be merged after these two ones:

Depends on #251 #308

### CODE REVIEW

File: docs/en/pid-eaa-data-model.rst [hunk 1]
```diff
@@ -22,9 +22,9 @@ The PID/(Q)EAA data format and the mechanism through which a digital credential
 SD-JWT
 ======
 
-The PID/(Q)EAA is issued in the form of a Digital Credential. The Digital Credential format is `Selective Disclosure JWT format <https://datatracker.ietf.org/doc/html/draft-ietf-oauth-selective-disclosure-jwt-04>`_ as specified in `[SD-JWT-based Verifiable Credentials 02] <https://www.ietf.org/archive/id/draft-ietf-oauth-sd-jwt-vc-02.html>`__.
+The PID/(Q)EAA is issued in the form of a Digital Credential. The Digital Credential format is `Selective Disclosure JWT format <https://datatracker.ietf.org/doc/html/draft-ietf-oauth-selective-disclosure-jwt-07>`_ as specified in `[SD-JWT-based Verifiable Credentials 02] <https://www.ietf.org/archive/id/draft-ietf-oauth-sd-jwt-vc-02.html>`__.
```
Comment (peppelinux) [inline docs/en/pid-eaa-data-model.rst]:
```suggestion
The PID/(Q)EAA is issued in the form of a Digital Credential. The Digital Credential format is `Selective Disclosure JWT format <https://datatracker.ietf.org/doc/html/draft-ietf-oauth-selective-disclosure-jwt-10>`_ as specified in `[SD-JWT-based Verifiable Credentials 02] <https://www.ietf.org/archive/id/draft-ietf-oauth-sd-jwt-vc-02.html>`__.
```

File: docs/en/pid-eaa-data-model.rst [hunk 2]
```diff
@@ -22,9 +22,9 @@ The PID/(Q)EAA data format and the mechanism through which a digital credential
...
 
-An SD-JWT is a JWT that MUST be signed using the Issuer's private key. The SD-JWT payload of the MUST contain the **_sd_alg** claim described in `[SD-JWT]. Section 5.1.2. <https://datatracker.ietf.org/doc/html/draft-ietf-oauth-selective-disclosure-jwt-04>`_ and other claims specified in this section, some of them may be selectively disclosable claims. 
+An SD-JWT is a JWT that MUST be signed using the Issuer's private key. The SD-JWT payload of the MUST contain the **_sd_alg** claim described in `[SD-JWT]. Section 5.1.2. <https://datatracker.ietf.org/doc/html/draft-ietf-oauth-selective-disclosure-jwt-07>`_ and other claims specified in this section, some of them may be selectively disclosable claims.
```
Comment (peppelinux) [inline docs/en/pid-eaa-data-model.rst]:
```suggestion
An SD-JWT is a JWT that MUST be signed using the Issuer's private key. The SD-JWT payload of the MUST contain the **_sd_alg** claim described in `[SD-JWT]. Section 5.1.2. <https://datatracker.ietf.org/doc/html/draft-ietf-oauth-selective-disclosure-jwt-10>`_ and other claims specified in this section, some of them may be selectively disclosable claims. 
```

File: docs/en/pid-eaa-data-model.rst [hunk 3]
```diff
@@ -42,7 +42,7 @@ The Disclosures are sent to the Holder together with the SD-JWT in the *Combined
 
   <Issuer-Signed-JWT>~<Disclosure 1>~<Disclosure 2>~...~<Disclosure N>
 
-See `[SD-JWT VC] <https://www.ietf.org/archive/id/draft-ietf-oauth-sd-jwt-vc-02.html>`_ and `[SD-JWT] <https://datatracker.ietf.org/doc/html/draft-ietf-oauth-selective-disclosure-jwt-04>`__ for more details. 
+See `[SD-JWT VC] <https://www.ietf.org/archive/id/draft-ietf-oauth-sd-jwt-vc-02.html>`_ and `[SD-JWT] <https://datatracker.ietf.org/doc/html/draft-ietf-oauth-selective-disclosure-jwt-07>`__ for more details.
```
Comment (peppelinux) [inline docs/en/pid-eaa-data-model.rst]:
```suggestion
See `[SD-JWT VC] <https://www.ietf.org/archive/id/draft-ietf-oauth-sd-jwt-vc-02.html>`_ and `[SD-JWT] <https://datatracker.ietf.org/doc/html/draft-ietf-oauth-selective-disclosure-jwt-10>`__ for more details. 
```

**Comment by fmarino-ipzs** [comment]
This PR is outdated by the PR #378. 
@peppelinux, if you agree we can close this PR.

**Pull request closed without merging**